vulture_os/toolkit/api_parser/sentinel_one_singularity_mobile/sentinel_one_singularity_mobile.py

```diff
@@ -0,0 +1,160 @@
+along with Vulture OS.  If not, see http://www.gnu.org/licenses/.
+"""
+__author__ = ""
+__credits__ = []
+__license__ = "GPLv3"
+__version__ = "4.0.0"
+__maintainer__ = "Vulture OS"
+__email__ = "contact@vultureproject.org"
+__doc__ = 'Sentinel One Mobile Zimperium API Parser toolkit'
+__parser__ = 'SENTINEL ONE MOBILE ZIMPERIUM'
+
+
+
+import requests
+import json
+import logging
+
+import datetime
+from django.conf import settings
+from django.utils import timezone
+from toolkit.api_parser.api_parser import ApiParser
+
+logging.config.dictConfig(settings.LOG_SETTINGS)
+logger = logging.getLogger('api_parser')
+
+
+class SentinelOneSingularityMobileParser(ApiParser):
+    HEADERS = {
+        "Accept": "application/json"
+    }
+    SIZE = 2000
+
+    def __init__(self, data):
+        super().__init__(data)
+
+        self.__access_token = ""
+        self.session = None
+
+        self.host = data["sentinel_one_singularity_mobile_host"]
+        self.client_id = data["sentinel_one_singularity_mobile_client_id"]
+        self.__secret = data["sentinel_one_singularity_mobile_client_secret"]
+
+    def get_access_token(self):
+        if not self.__access_token:
+            try:
+                url = f"https://{self.host}/api/auth/v1/api_keys/login"
+                data = {
+                    "clientId": self.client_id,
+                    "secret": self.__secret,
+                }
+                response = requests.post(url, json=data, headers=self.HEADERS)
+                response.raise_for_status()
+            except requests.exceptions.HTTPError as err:
+                logger.debug(err, extra={'frontend': str(self.frontend)})
+                exit(0)
+            else:
+                response_json = response.json()
+                self.__access_token = response_json["accessToken"]
+        return self.__access_token
+
+    def _connect(self):
+        if self.session is None:
+            self.session = requests.Session()
+            self.session.headers.update(self.HEADERS)
+            self.session.headers.update({"Authorization": f"Bearer {self.get_access_token()}"})
+        return True
+
+    def get_logs(self, since, to):
+        if self.session is None:
+            self._connect()
+
+        def convert(x): return x.strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3] + 'Z'
+        url = f"https://{self.host}/api/threats/public/v1/threats"
+
+        params = {
+            "page": 0,
+            "size": self.SIZE,
+            "after": convert(since),  # >=
+            "before": convert(to),    # <=
+            "sort": "timestamp,asc",
+            "module": "ZIPS"
+        }
+
+        response = self.session.get(url, params=params)
+
+        resp = response.json()
+        logger.debug(f"url: {url}\nparams: {params}", extra={'frontend': str(self.frontend)})
+
+        if "content" not in resp:
+            raise Exception(str(resp))
+        return resp['content']
```

Comment:
```suggestion
        logger.debug(f"url: {url}\nparams: {params}", extra={'frontend': str(self.frontend)})

        return response.json()['content']
```
an exception will already be raised if the "content" object is missing